VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.component.html

```diff
@@ -7,9 +7,12 @@
   </div>
   <div class="govuk-grid-column-full">
     <p id="tooltip-hearing-role-{{ participant.id }}" class="govuk-!-margin-bottom-1">
-      {{ participant.hearingRole }}
+      {{this.getHearingRole()}}
```

Comment:
Just a small question. I have see some code samples using the pipe to translate and some using the Transalation service like in this PR. Is there a preference or standard here as to which approach is taken and when?

Reply:
Just to highlight what was discussed in a seperate chat, the preference here is to use the pipes over using the translate service itself